plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeLegacyCreateTableWithExistingLocation.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.Session;
+import io.trino.plugin.hive.metastore.HiveMetastore;
+import io.trino.plugin.hive.metastore.file.FileHiveMetastore;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.QueryRunner;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static com.google.common.io.MoreFiles.deleteRecursively;
+import static com.google.common.io.RecursiveDeleteOption.ALLOW_INSECURE;
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestDeltaLakeLegacyCreateTableWithExistingLocation
+        extends AbstractTestQueryFramework
+{
+    private static final String CATALOG_NAME = "delta_lake";
+
+    private File dataDirectory;
+    private HiveMetastore metastore;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        this.dataDirectory = Files.createTempDirectory("test_delta_lake").toFile();
+        this.metastore = FileHiveMetastore.createTestingFileHiveMetastore(dataDirectory);
+
+        return DeltaLakeQueryRunner.createDeltaLakeQueryRunner(
+                CATALOG_NAME,
+                ImmutableMap.of(),
+                ImmutableMap.of(
+                        "delta.unique-table-location", "true",
+                        "hive.metastore", "file",
+                        "hive.metastore.catalog.dir", dataDirectory.getPath()));
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void tearDown()
+            throws IOException
+    {
+        if (metastore != null) {
+            metastore.dropDatabase("tpch", false);
+            deleteRecursively(dataDirectory.toPath(), ALLOW_INSECURE);
+        }
+    }
+
+    @Test
+    public void testLegacyCreateTable()
+    {
+        String tableName = "test_legacy_create_table_" + randomNameSuffix();
+
+        assertQuerySucceeds(format("CREATE TABLE %s AS SELECT 1 as a, 'INDIA' as b, true as c", tableName));
+        assertThat(query("SELECT * FROM " + tableName))
+                .matches("VALUES ROW(INT '1', VARCHAR 'INDIA', BOOLEAN 'true')");
```

Comment:
Remove redundant ROW and avoid redundant string literals. By the way, you can use `assertQuery` for simple query assertions. Please fix L91 as well. 
```java
assertQuery("SELECT * FROM " + tableName, "VALUES (1, 'INDIA', true)");
```

Reply:
Missed changing this file in the earlier commit. Updated now.